scripts/deploy_contracts.js

```diff
@@ -29,47 +29,27 @@ async function exec() {
   await initWeb3()
 
   // Retrieve deployment data for the active network.
-  const path = `./gamedata/deploy.${constants.ACTIVE_NETWORK.name}.json`;
-  try {
-    deployData = JSON.parse(fs.readFileSync(path, 'utf8'));
-  }
-  catch(err){
-    deployData = {};
-  }
+  const deployData = loadDeployData(DEPLOY_DATA_PATH)
 
   // Determine which contracts need to be deployed.
   let count = 0;
-  const deployedKey = `deployed_${constants.ACTIVE_NETWORK.name}`
   if(needsDeploy(deployData.ethernaut)) {
-    count++
-    console.log(colors.red(`(${count}) Will deploy Ethernaut.sol!`))
+    console.log(colors.red(`(${++count}) Will deploy Ethernaut.sol!`))
```

Comment:
I don't like this change. It seems to me that the count increase is now hidden on a UI/debug statement.

Reply:
Hm, but `count` is used **only** in the UI/debug statements - we don't use it for anything else. 

I can revert this one, but please confirm.

Reply:
It's just a nit, so my personal policy is that if you disagree, you get to choose because you are doing the work.
I still prefer it as before, because when it is defined, it is not clear that it's only for UI statements. It is not defined in an isolated UI component, so I prefer it's use to be as clear as possible. 

But, your call.

Reply:
Alright, I put it back